pkg/bucketeer/event/processor.go

```diff
@@ -3,10 +3,16 @@ package event
 
 import (
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/ca-dp/bucketeer-go-server-sdk/pkg/bucketeer/log"
+	"github.com/ca-dp/bucketeer-go-server-sdk/pkg/bucketeer/uuid"
+	protoevent "github.com/ca-dp/bucketeer-go-server-sdk/proto/event/client"
 	protofeature "github.com/ca-dp/bucketeer-go-server-sdk/proto/feature"
 	protouser "github.com/ca-dp/bucketeer-go-server-sdk/proto/user"
+	"github.com/golang/protobuf/ptypes"
```

Comment:
Please separate the third-party libraries from the bucketeer's one.

Reply:
I'm going to fix this, but it's not a Go's standard formatting that the `gofmt` does.
If apply this rule, we should use the `goimports` with the local flag in Makefile instead of `gofmt`.

I created the issue: https://github.com/ca-dp/bucketeer/issues/2647